apps/frontend/src/renderer/components/EnvConfigModal.tsx

```
@@ -498,29 +501,29 @@ export function EnvConfigModal({
                           <Copy className="h-3 w-3 ml-1" />
                         </button>
                       </li>
-                      <li>Copy the token and paste it below</li>
+                      <li>{t('envConfig.step3CopyToken')}</li>
                     </ol>
                     <button
                       onClick={handleOpenDocs}
                       className="text-info hover:text-info/80 flex items-center gap-1 mt-2"
                     >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)